keps/sig-instrumentation/0034-distributed-tracing-kep.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Leveraging Distributed Tracing to Understand Kubernetes Object Lifecycles
+authors:
+  - "@Monkeyanator"
+editors:
+  - "@dashpole"
+owning-sig: sig-instrumentation
```

Comment:
Can you use the `participating-sigs:` to add sig-architecture.

Reply:
done